### ISSUE TITLE
Expected values of `ins_prob` and `ins_loss` in MoCo when training is working

### ISSUE THREAD

**Issue by nlml** [HobbitLong/CMC#38]
Hi there

Thanks a lot for this great repo!

I am trying out MoCo on my own dataset (I also added additional augmentations). Training appears to have converged, but the max value I get for `ins_prob` is about `13.35`, and the lowest value I get for loss is about `0.2422`.

I am wondering what metrics you got when training on Imagenet? Am not sure what a "good" score should look like.

Here are screenshots from training progress in tensorboard (ignore the multiple lines at the start of training).

![image](https://user-images.githubusercontent.com/12813713/73645839-60aa1300-4678-11ea-8abf-d2f69bb229c6.png)

![image](https://user-images.githubusercontent.com/12813713/73645847-66075d80-4678-11ea-80c5-c60d68cbeac3.png)

Thanks,
Liam

**Comment by chigur** [comment]
I think a simple solution would be to use the pre-trained MoCo model and get the loss for Imagenet. That will give you the final loss achieved by the model.
